### DISCUSSION TITLE
How can the Meshlets gem be used in editor?

### DISCUSSION THREAD

**Discussion by ohmaya** [o3de/o3de#12810] [Help Wanted]
Can only showcase the rendering in Atom Sample Viewer (https://github.com/o3de/o3de/tree/development/Gems/Meshlets/ASV_GPU_and_CPU_Demo)?

**Comment by santorac** [comment]
The meshlets gem is a proof-of-concept, I'm not sure to what extent it can be used in the Editor.  I've only ever seen it working in AtomSampleViewer. @adiblev is the original developer, perhaps he can provide some guidance.
